### ISSUE TITLE
Naming Issue on .profile - User Folder in Windows

### ISSUE THREAD

**Issue by tlin623** [michaeltreat/Windows-Subsystem-For-Linux-Setup-Guide#4]
Hi Michael,

Thank you for this wonderful guide to WSL.   I was wondering if you've encountered the issue on adding space to the user folder under the .profile file.  

For example, 

`export wr=~/../..mnt/c/Users/FirstName LastName/`

I've tried googling the issue, some people suggested forward slash \ to handle the spacing issue, but that does not work for me.  I've also tried, single quote or double quote.

Example:

```
export wr=~/../..mnt/c/Users/FirstName\ LastName/
export wr="~/../..mnt/c/Users/FirstName LastName/"
export wr='~/../..mnt/c/Users/FirstName LastName/'
```

Sincerely, thanks again for your great work.

**Comment by michaeltreat** [comment]
Hey @tlin623 , thanks for the compliments on the guide! 

This is my bad. The issue is that we should be de-referencing the bash variable when using it, and we should be doing that wether it's a path with a space or not. I will go in and update that line in the guide!

To de-reference a variable, you invoke it with double quotes around it.

Here's an example:

**Incorrect**:
If you type ` $wr` then you'll see the error : no such file or directory

**Correct**
But, if you type `"$wr"` then bash will tell you it's a directory.

Note the double quotes around this variable when you call it. That's called de-referencing the variable.

**Solution**
Your code should look like this:

```
export wr=~/../../mnt/c/Users/FirstName\ LastName/
alias cdwr='cd "$wr"'
```

Note the backslash escape character to escape the space between your first and last name.
Also note the double quotes around $wr

This is the link to [StackOverflow](https://stackoverflow.com/questions/43787476/how-to-add-path-with-space-in-bash-variable) where I found the answer.

I tested this on my machine and it works, but let me know if it works for you! 

Thanks again!
